datahub-web-react/src/app/recommendations/renderer/component/EntityNameList.tsx

```diff
@@ -38,18 +38,27 @@ const ThinDivider = styled(Divider)`
     margin: 0px;
 `;
 
+type AdditionalProperties = {
+    path?: Entity[];
+};
+
 type Props = {
+    // additional data about the search result that is not part of the entity used to enrich the
+    // presentation of the entity. For example, metadata about how the entity is related for the case
+    // of impact analysis
+    additionalPropertiesList?: Array<AdditionalProperties>;
```

Comment:
One alternate is to take not an array of <Entity> but an array of <EntityWithProperties> or something - did u consider this path?

Reply:
This whole array index based correlation makes me nervous

Reply:
will add validation that arrays are same size. we'll keep this format for now since it just has one user and is easy to back out of